vite.config.ts

```diff
@@ -1,7 +1,13 @@
-import { defineConfig } from 'vite'
+/// <reference types="vitest" />
 import react from '@vitejs/plugin-react-swc'
+import { defineConfig } from 'vite'
 
 // https://vitejs.dev/config/
 export default defineConfig({
   plugins: [react()],
+  test: {
+    globals: true,
```

Comment:
This option make `globals` work, but if TypeScript complains, remember to add the following to your `tsconfig.json`:
```json
"types": ["vitest/globals"],
```